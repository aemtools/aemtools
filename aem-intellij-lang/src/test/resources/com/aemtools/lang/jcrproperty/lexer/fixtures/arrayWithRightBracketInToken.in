[/some/path/([a-z]+_[A-Z]+)=/some/path]
