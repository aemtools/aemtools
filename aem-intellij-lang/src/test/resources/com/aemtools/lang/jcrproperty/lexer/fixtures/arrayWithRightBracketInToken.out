[ ('[')
ARRAY_VALUE_TOKEN ('/some/path/([a-z]+_[A-Z]+)=/some/path')
] (']')
